_rules/inline-link-in-paragraph-has-distinguishable-style-on-hover-and-focus-548868.md

```diff
@@ -0,0 +1,318 @@
+---
+id: 548868
+name: Inline link has distinguishable style or content on hover and focus
+rule_type: atomic
+description: |
+  This rule checks that inline links are distinguishable from the surrounding text through a difference in style or content when the links are hovered or focused.
+accessibility_requirements:
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Carlos Duarte
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any [visible][] HTML element that is a [semantic link][], for which all the following is true:
+
+- **link text**: the element has [visible][] [text nodes][text node] as [descendants][descendant] in the [flat tree][]; and
+- **non-link line text**: the element is [rendered on a line][] containing [visible][] [text nodes][text node] that are not [descendants][descendant] in the [flat tree][] of a [semantic link][]; or
+- **different color**: the element's [foreground color][] and the [foreground color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1, or the element's [background color][] and the [background color][] of the **non-link line text** elements have a [highest possible contrast][] that is at least 3.0:1.
+
+## Expectation
+
+For each test target, at least one of the following is true when it is [focused][] or [hovered][]:
```

Comment:
The phrase "when it is [focused][] or [hovered] is placed sometimes in the middle of the sentence, some others at the end. I think we need to place it consistently either at the end or in the middle of the sentences when it appears in each of the rules.
"